bin/check-lv-usage.rb

```diff
@@ -73,7 +73,7 @@ class CheckLVUsage < Sensu::Plugin::Check::CLI
   # Setup variables
   #
   def initialize(argv = ARGV)
-    super
+    super(argv)
```

Comment:
@insom I needed the `argv` as I use it in my test cases....which I forgot to include in the original PR, sorry about that.